fre/fre.py

```diff
@@ -21,7 +21,8 @@
                        "yamltools": ".yamltools.freyamltools.yamltools_cli",
                        "make": ".make.fremake.make_cli",
                        "app": ".app.freapp.app_cli",
-                       "cmor": ".cmor.frecmor.cmor_cli"                      },
+                       "cmor": ".cmor.frecmor.cmor_cli",
+                       "analysis": ".analysis.freanalysis.analysis_cli"},
```

Comment:
another thing i doubt is the source of the current failures, but `python` is never a big fan of tabs, and this looks like a spurious tab got in here somehow!

Reply:
wait- this might have been a text-on-my-screen rendering thing. possibly not an issue, but also not impossible github doesnt render tabs everywhere with the same conversion rule.